eth/downloader/downloader_test.go

```diff
@@ -1215,7 +1205,6 @@ func testBlockHeaderAttackerDropping(t *testing.T, protocol int) {
 		{errBadPeer, true},            // Peer was deemed bad for some reason, drop it
 		{errStallingPeer, true},       // Peer was detected to be stalling, drop it
 		{errNoPeers, false},           // No peers to download from, soft race, no issue
-		{errPendingQueue, false},      // There are blocks still cached, wait to exhaust, no issue
 		{errTimeout, true},            // No hashes received in due time, drop the peer
 		{errEmptyHashSet, true},       // No hashes were returned as a response, drop as it's a dead end
 		{errEmptyHeaderSet, true},     // No headers were returned as a response, drop as it's a dead end
```

Comment:
Maybe add these two to the end of the test suite. Just noticed they are missing:

```
		{errCancelReceiptFetch, false}, // Synchronisation was canceled, origin may be innocent, don't drop
		{errCancelStateFetch, false},   // Synchronisation was canceled, origin may be innocent, don't drop
```